downloads.html

```diff
@@ -99,7 +99,11 @@ <h3>Packages</h3>
               <a href="https://launchpad.net/~minetestdevs/+archive/ubuntu/daily-builds">Daily PPA</a>
           </li>
           <li>Debian -
-              <a href="https://packages.debian.org/search?keywords=minetest">Stable</a></li>
+              use
+              <a href="https://flathub.org/apps/details/net.minetest.Minetest">Flatpak</a>
+              or compile from
+              <a href="https://github.com/minetest/minetest/tree/stable-5">source</a>,
```

Comment:
rather big usability issue: if you read the docs in the stable version of the repo the docs give you the git commands to download from master. it's not even mentioned that that's not the version you might want

Reply:
This is already an issue with the "source" section of the downloads page

Reply:
Sure but maybe we shouldn't so prominently be linking it then. Remove the `<a>` from "source"?